server/src/api/modules/groups/group.service.js

```diff
@@ -255,10 +258,13 @@ async function edit(id, name, verificationCode, members) {
 
   if (members) {
     // Check if every username in the list is valid
-    for (let i = 0; i < members.length; i += 1) {
-      if (!playerService.isValidUsername(members[i].username)) {
-        throw new BadRequestError(`Invalid player username: ${members[i].username}`);
-      }
+    const usernamesCheck = await playerService.isValidUsernames(members);
```

Comment:
Maybe this variable could be renamed to "invalidUsernames"?

**Also: isValidUsernames is not an async function so it does not need the "await" before calling it.**

Reply:
Same as above.